README.rst

```diff
@@ -344,3 +345,15 @@ You can use Apache Airflow to author, schedule and monitor SageMaker workflow.
 For more information, see `SageMaker Workflow in Apache Airflow`_.
 
 .. _SageMaker Workflow in Apache Airflow: https://sagemaker.readthedocs.io/en/stable/using_workflow.html
+
+SageMaker Autopilot
+-------------------
+
+Amazon SageMaker Autopilot is an automated machine learning (commonly referred to as AutoML) solution for tabular
```

Comment:
I would move "(commonly referred to as AutoML)" to after "solution". Also, put "AutoML" in quotes

Reply:
Done